Source/UINavigation/Private/UINavWidget.cpp

```diff
@@ -717,9 +717,24 @@ void UUINavWidget::NativeTick(const FGeometry & MyGeometry, float DeltaTime)
 {
 	Super::NativeTick(MyGeometry, DeltaTime);
 
-	if (IsSelectorValid() && bMovingSelector)
+	if (IsSelectorValid())
 	{
-		HandleSelectorMovement(DeltaTime);
+		if (bUpdateSelector)
+		{
+			if (UpdateSelectorWaitForTick == 1)
```

Comment:
Change this to >= 1

Reply:
I did the change in 55cd0b1, but as bUpdateSelector becomes false when UpdateSelectorWaitForTick is 1 it is not going to happen. However you made me think on race conditions, for those to happen:
1- Multiple selector updates at the same time: as updates come from user input moving mouse/d-pad this is very unlikely to happen. If it happens only one of the two will happen.
2- NativeTick happens at the same time as handling of user input: No idea how the engine works. Ideally the nativeTick function can't be interrupted by an scheduler, as it must end before next frame is drawn, but we have multicore. If not we need to block code triggered by user input (the code setting bUpdateSelector and UpdateSelectorWaitForTick to true) while tick is executing with a mutex or so . 

I tested it and works properly.

Reply:
I have read that you can define individual actors that update while the game is paused.

What you could do, is to exclude an actor (widget) from being paused. That way we can still implement #135, what do you think?

Reply:
I am wondering if this means that we also need to add "bAlwaysTick=true" or "settickable while paused" for this solution to work, or if this also fixes the timers approach or there is another way of executing timers while paused.

Reply:
That would be great 😄 thanks for being thorough! 💪